<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#StartDialog{
background-image: url(:/TestMode/startBack.jpg);}
QWidget{background-color: rgba(0, 0, 0, 0);}
QLabel{font: 75 24pt &quot;Adobe Arabic&quot;;
color: rgb(255, 0, 0);}
QPushButton{font: 75 24pt &quot;Adobe Arabic&quot;;
color: rgb(0, 255, 255);}
QPushButton:hover{font: 75 24pt &quot;Adobe Arabic&quot;;
color: rgb(255, 255, 0);}</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>545</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="whatsThis">
    <string>退出</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/TestMode/exit0.png);}
QPushButton:hover{
border-image: url(:/TestMode/exit1.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>441</width>
     <height>527</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="3,5">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <item row="5" column="0">
     <widget class="QPushButton" name="button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第六关</string>
      </property>
      <property name="whatsThis">
       <string>开始第六关</string>
      </property>
      <property name="text">
       <string>第六关</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第七关</string>
      </property>
      <property name="whatsThis">
       <string>开始第七关</string>
      </property>
      <property name="text">
       <string>第七关</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第二关</string>
      </property>
      <property name="whatsThis">
       <string>开始第二关</string>
      </property>
      <property name="text">
       <string>第二关</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第三关</string>
      </property>
      <property name="whatsThis">
       <string>开始第三关</string>
      </property>
      <property name="text">
       <string>第三关</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第九关</string>
      </property>
      <property name="whatsThis">
       <string>开始第九关</string>
      </property>
      <property name="text">
       <string>第九关</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第八关</string>
      </property>
      <property name="whatsThis">
       <string>开始第八关</string>
      </property>
      <property name="text">
       <string>第八关</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="button10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第十关</string>
      </property>
      <property name="whatsThis">
       <string>开始第十关</string>
      </property>
      <property name="text">
       <string>第十关</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第四关</string>
      </property>
      <property name="whatsThis">
       <string>开始第四关</string>
      </property>
      <property name="text">
       <string>第四关</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第一关</string>
      </property>
      <property name="whatsThis">
       <string>开始第一关</string>
      </property>
      <property name="text">
       <string>第一关</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>162</width>
        <height>47</height>
       </size>
      </property>
      <property name="toolTip">
       <string>开始第五关</string>
      </property>
      <property name="whatsThis">
       <string>开始第五关</string>
      </property>
      <property name="text">
       <string>第五关</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label2">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label1">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label4">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label5">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label3">
      <property name="text">
       <string>label3</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label8">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label7">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label6">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label10">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label9">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="loadAiBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>545</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>载入AI</string>
   </property>
   <property name="whatsThis">
    <string>载入AI</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-image: url(:/TestMode/openAi0.png);}
QPushButton:hover{
border-image: url(:/TestMode/openAi1.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="aiPathEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>555</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 200);</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="test_TestMode.qrc"/>
 </resources>
 <connections/>
</ui>
